scripts/pipelines/gcloud/templates/deploy/deploy-pipeline.yml.template

```
@@ -0,0 +1,58 @@
+steps:
+
+- name: ubuntu
+  id: Get Credentials
+  entrypoint: bash
+  secretEnv: ['registry_credentials']
+  args:
+    - -c
+    - echo $$registry_credentials > credentials.env
```

```suggestion
- name: gcr.io/cloud-builders/gcloud
  id: Get credentials
  entrypoint: bash
  secretEnv: ['registry_credentials']
  args:
    - -c
    - echo $$registry_credentials > credentials.env
```

Better use already used image